### PR TITLE
Fix #1990: Deduplicate % only for Node.js lower than v2.1.0.

### LINKED ISSUE #1990: Node.js 4's console does not deduplicate % anymore -> our test breaks
Failing build log: https://scala-webapps.epfl.ch/jenkins/job/scalajs-task-worker/66687/console

We need to remove the hack at https://github.com/scala-js/scala-js/blob/6e3255a6c0e502c317b9d14a645dcb091a3407aa/js-envs/src/main/scala/org/scalajs/jsenv/nodejs/NodeJSEnv.scala#L276-L296 to get back a normal behavior.

I don't know whether we can have a "smart hack" that duplicates %'s on old Nodes, and doesn't on recent ones.

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

**Comment by nicolasstucki** [comment]
Review by @sjrd.

File: js-envs/src/main/scala/org/scalajs/jsenv/nodejs/NodeJSEnv.scala [hunk 1]
```diff
@@ -279,18 +279,21 @@ class NodeJSEnv private (
...
-              var args = arguments;
-              if (args.length >= 1 && args[0] !== void 0 && args[0] !== null) {
-                args[0] = args[0].toString().replace(/%/g, "%%");
-              }
-              oldLog.apply(console, args);
-            };
-            console.log = newLog;
-          })();
+          if (process.version.startsWith("v0.") || process.version.startsWith("v1.") ||
+              process.version.startsWith("v2.0")) {
```
Comment (sjrd) [inline js-envs/src/main/scala/org/scalajs/jsenv/nodejs/NodeJSEnv.scala]: `v2.0.` with a trailing `.` ;)

**Comment by sjrd** [comment]
That's all.

**Comment by sjrd** [comment]
LGTM

**Comment by sjrd** [comment]
retest this please

**Comment by sjrd** [comment]
This causes other problems. I'm filing another fix.

**Pull request closed without merging**